bliss/models/decoder.py

```diff
@@ -471,11 +464,10 @@ def __init__(self, tile_slen, ptile_slen, n_bands, galaxy_model: GalaxyModel):
             raise TypeError("galaxy_model is not a valid type.")
         self.galaxy_decoder = galaxy_decoder
 
-    def forward(self, locs, galaxy_params, galaxy_bools):
+    def forward(self, galaxy_params: Tensor, galaxy_bools: Tensor):
         """Renders galaxy tile from locations and galaxy parameters."""
         # max_sources obtained from locs, allows for more flexibility when rendering.
-        n_ptiles = locs.shape[0]
-        max_sources = locs.shape[1]
+        n_ptiles, max_sources, _ = galaxy_bools.shape
         n_galaxy_params = galaxy_params.shape[-1]
 
         galaxy_params = galaxy_params.view(n_ptiles, max_sources, n_galaxy_params)
```

Comment:
some of these assertion can be removed I think since they are redundant with the `.view`

Reply:
(not part of your PR so feel free to ignore this suggestion)

Reply:
Yes let's simplify the asserts below